internal/cmd/beta/server/log/log.go

```diff
@@ -0,0 +1,138 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverIdArg = "SERVER_ID"
+
+	lengthLimitFlag    = "length"
+	defaultLengthLimit = 2000 // lines
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ServerId string
+	Length   *int64
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "log",
+		Short: "Get server console log",
+		Long:  "Get server console log.",
+		Args:  args.SingleArg(serverIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get server console log for the server with ID "xxx"`,
+				"$ stackit beta server log xxx",
+			),
+			examples.NewExample(
+				`Get server console log for the server with ID "xxx" and limit output lines to 1000`,
+				"$ stackit beta server log xxx --length 1000",
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			serverLabel, err := iaasUtils.GetServerName(ctx, apiClient, model.ProjectId, model.ServerId)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get server name: %v", err)
+				serverLabel = model.ServerId
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("server log: %w", err)
+			}
+
+			log := *resp.GetOutput()
+			lines := strings.Split(log, "\n")
+			p.Info("lines = %d", len(lines))
+
+			if len(lines) > int(*model.Length) {
+				// Truncate output and show most recent logs
+				start := len(lines) - int(*model.Length)
+				p.Info("Log for server %q\n%s", serverLabel, strings.Join(lines[start:], "\n"))
+				return nil
+			}
```

Comment:
This output should be handled in an `outputResult` function (similar to other commands), to account for the different output formats: JSON, YAML, or "pretty"

Reply:
In "pretty" we are free to display however we want, does not necessarily need to be a table